docs/developer/README.md

```diff
@@ -166,6 +166,16 @@ from .common import less_console_noise
             # <test code goes here>
 ```
 
+### Accessibility Testing in the browser
+
+We use the [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) browser extension 
+from ssa.gov for accessibility testing outside the pipeline. 
+
+ANDI will get blocked by our CSP settings, so you will need to install the 
+[Disable Content-Security-Policy extension](https://chrome.google.com/webstore/detail/disable-content-security/ieelmcmcagommplceebfedjlakkhpden) and activate it for the page you'd like
```

Comment:
okay i downloaded the extension and it looks like its activating and then I click andi but still don't see ANDI popping up correctly

Reply:
Try refreshing with dev tools open > network > no cache, click on ANDI again.

Reply:
I thought I replied to this last night, I tried this. I don't see "no cache" but i clicked "disable cache", refreshed and ran again. I still don't see ANDI opening

Reply:
Yeah I already refreshed after disabling the cahce and went ahead and cleared the cache. Still not working